Sources/NIOCore/IO.swift

```diff
@@ -14,7 +14,21 @@
 
 #if os(Windows)
 import ucrt
+import func WinSDK.FormatMessageW
+import func WinSDK.LocalFree
+import let WinSDK.FORMAT_MESSAGE_ALLOCATE_BUFFER
+import let WinSDK.FORMAT_MESSAGE_FROM_SYSTEM
+import let WinSDK.FORMAT_MESSAGE_IGNORE_INSERTS
+import let WinSDK.LANG_NEUTRAL
+import let WinSDK.SUBLANG_DEFAULT
 import typealias WinSDK.DWORD
+import typealias WinSDK.WCHAR
+import typealias WinSDK.WORD
+
+@_transparent
```

Comment:
Can we remove `@_transparent`? It's fine if this generates a function call and I'd prefer we didn't use attributes the Swift team might rename if we can possibly avoid it.

Reply:
I suppose; it is only there to deal with limitations of the clang importer.  This is a function-like macro, just that it cannot be imported by the swift compiler.